frontend/components/homepage/dashboard.tsx

```diff
@@ -1,26 +1,61 @@
-import { Box, Center, Flex, Heading } from "@chakra-ui/react";
+import { Box, Center, Flex, Heading, ListItem, UnorderedList } from "@chakra-ui/react";
 import Card from "./card";
 import { links, link } from "@frontend/components/homepage/dashboard-structure";
 import { useSession } from "next-auth/react";
+import Chart from "@frontend/components/homepage/line-chart";
+import { USER_ROLES } from "@frontend/utils/constants";
+import { StatusDataProps } from "@frontend/models/patient";
 
-export default function Dashboard() {
+export default function Dashboard({ data }: { data: StatusDataProps[] }) {
     const { data: session } = useSession();
     const userRole = session?.user?.Role;
     const userName = session?.user?.FirstName + " " + session?.user?.LastName;
-    // depending on the role, will render different box for different functionalities
+
+    // reverse data array for line chart
+    data.reverse();
+
     return (
-        <Box>
+        <>
             <Heading paddingLeft={"50px"}>
                 Welcome {userRole} {userName}
             </Heading>
-            <Center marginTop={"5%"}>
-                <Flex flexWrap="wrap">
-                    {links.map(({ label, url, roleRequired, img }: link) => {
-                        const renderCard = roleRequired === userRole || !roleRequired;
-                        return renderCard && <Card key={label} label={label} image={img} url={url} />;
-                    })}
-                </Flex>
-            </Center>
-        </Box>
+            <Flex flexWrap={"wrap"}>
+                {/* Patient dashboard */}
+                {session?.user?.Role === USER_ROLES.patient ? (
```

Comment:
can be changed to user session?.user?.Role === USER_ROLES.patient and remove the ? and :<></> at the end